content/docs/installation.md

```diff
@@ -4,15 +4,129 @@ description: Get up and running on Linux and a variety of container platforms
+
+`SYSDIG_PROBE_URL` - Set this environment variable for the Falco pod to override the default host for prebuilt kernel modules. This should be only the host portion of the URL without the trailing slash, ie `https://myhost.mydomain.com`. The kernel modules should be placed in directory `/stable/sysdig-probe-binaries/` and named as follows:
+`falco-probe-${falco_version}-$(uname -i)-$(uname -r)-{md5sum of kernel config}.ko`
+
+The `falco-probe-loader` script will name the module in this format by default. 
+
+### Helm
+
+Helm is one of the preferred methods for installing Falco on Kubernetes. The [Falco Helm chart](https://github.com/helm/charts/tree/master/stable/falco) provides an extensive set of [configuration values](https://github.com/helm/charts/tree/master/stable/falco#configuration) to start Falco with a variety of different configurations.
+
+Assuming you have Helm installed and running in your cluster, to deploy Falco with the default configuration simply run:
+```shell
+helm install --name falco stable/falco
+```
+
+To remove Falco from your cluster run:
+```
+helm delete falco
+```
+
+#### Kubernetes Response Engine
+
+Using the Falco Helm chart is the easiest way to deploy the [Falco Kuberentes Response Engine (KRE)](https://github.com/falcosecurity/kubernetes-response-engine). The KRE provides the ability to send Falco alerts to a messaging service such as NATS, AWS SNS, or Google Pub/Sub. This allows Falco alerts to be processed by subcribers of the respective messaging services. Refer to the `integrations.*` [configuration options](https://github.com/helm/charts/tree/master/stable/falco#configuration) of the Helm chart to enable this integration. 
+
+The KRE also allows you to deploy security playbooks (via serverless functions) that can take action when Falco rules are violated. Refer to the [Response Engine documentation](https://github.com/falcosecurity/kubernetes-response-engine/tree/master/playbooks) on how to deploy the included playbooks.
+
+### DaemonSet Manifests
+To run Falco as a Kubernetes [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/), follow the instructions below. These are the `generic` instructions for Kubernetes. See below for Kubernetes platform specific instructions.
+
+1. Clone the [Falco repo](https://github.com/falcosecurity/falco/) and change to the directory with the manifests.
+```shell
+git clone https://github.com/falcosecurity/falco/
+cd falco/integrations/k8s-using-daemonset
+```
+2. Create a service account and RBAC permissions which Falco will run as. This will allow Falco to connect to the Kubernetes API server to fetch resource metadata. 
+```shell
+kubectl apply -f k8s-with-rbac/falco-account.yaml
+```
+3. Create a service for the Falco pods. This will allow Falco to recieve [Kubernetes Audit Log Events](event-sources/kubernetes-audit). If you're not planning on using this feature, you can skip this step.
```

Comment:
```suggestion
3. Create a service for the Falco pods. This will allow Falco to receive [Kubernetes Audit Log Events](event-sources/kubernetes-audit). If you're not planning on using this feature, you can skip this step.
```